### ISSUE TITLE
Warn excluding attribute - not found in Terraform schema   

### ISSUE THREAD

**Issue by zioproto** [sl1pm4t/k2tf#19]
This example yaml file will generate a false positive warning:
```
---
apiVersion: v1
kind: Namespace
metadata:
  name: cert-manager
  labels:
    certmanager.k8s.io/disable-validation: "true"
```
How to reproduce:
```
cat 05-cert-manager.yaml | ./k2tf
10:12PM Warn | excluding attribute - not found in Terraform schema           attr=kubernetes_namespace.spec field=Namespace.Spec name=cert_manager type=kubernetes_namespace
resource "kubernetes_namespace" "cert_manager" {
  metadata {
    name = "cert-manager"

    labels {
      "certmanager.k8s.io/disable-validation" = "true"
    }
  }
}
```

The translation looks correct, I dont understand if the warning is a false positive.

Removing the line `kind: Namespace` fixes the warning.

**Comment by sl1pm4t** [comment]
Hi @zioproto I agree this is a confusing warning message, since the source YAML does not include a spec block.

The way k2tf converts the YAML to HCL, is by first using the Kubernetes Go libraries to create an object from the YAML definition. The tool then uses reflection to walk the structure of the Kubernetes go object, and creates the HCL structure based on the Kubernetes object.
In this case the Kubernetes library is creating a Namespace object with a Spec field, even though it's not defined in your YAML. The `k2tf` tool is then attempting to convert the Namespace.Spec field to a Terraform HCL resource sub-block and finding that it's not supported by the terraform-provider-kubernetes schema and outputting the warning. 

I'm working on a fix, that would prevent this warning being displayed, unless the `spec` block is explicitly defined in the YAML, in which case it's valid because the Terraform schema doesn't support it.

**Comment by sl1pm4t** [comment]
Closed by #20
